src/snuffleupagus.c

```diff
@@ -5,7 +5,6 @@
 #endif
 
 static PHP_INI_MH(OnUpdateConfiguration);
-static inline void sp_op_array_handler(zend_op_array *op);
```

Comment:
What's the problem with having a prototype? It'll ensure that any function declared before `sp_op_array_handler`'s implementation can use it. It's even required by [MISRA](https://www.synopsys.com/software-integrity/static-analysis-tools-sast/misra.html), 8.1

Reply:
`sp_op_array_handler` is defined on line 22 (21 with this patch) and used (the only usage) on line 52(51).